website/docs/r/emr_instance_group.html.markdown

```diff
@@ -32,10 +32,14 @@ The following arguments are supported:
 
 * `name` (Required) Human friendly name given to the instance group. Changing this forces a new resource to be created.
 * `cluster_id` (Required) ID of the EMR Cluster to attach to. Changing this forces a new resource to be created.
+* `instance_role` (Optional) The EC2 instance role for all instances in the instance group. Valid values are MASTER, CORE, TASK. Default `"TASK"`
 * `instance_type` (Required) The EC2 instance type for all instances in the instance group. Changing this forces a new resource to be created.
+* `bid_price` - (Optional) If set, the bid price for each EC2 instance in the instance group, expressed in USD. By setting this attribute, the instance group is being declared as a Spot Instance, and will implicitly create a Spot request. Leave this blank to use On-Demand Instances. `bid_price` can not be set for the `MASTER` instance group, since that group must always be On-Demand
```

Comment:
If we won't support `MASTER` instance groups, the extra verbiage can be removed here and above. 👍